python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -0,0 +1,186 @@
+#
+# This file is part of meas_extensions_trailedSources.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+r"""Naive trailed source measurement plugin. Measures the length, angle from
++x-axis, and end points of an extended source using the second moments.
+This measurement plugin aims to utilize the already measured shape second
+moments to naively estimate the length and angle, and thus end point, of a
+fast-moving, trailed source. A naive estimate for the trail length is obtained
+by doubling the semi-major axis, a, of the ellipse defined by the second
+moments. The angle, :math:`\Theta`, from the x-axis is computed similarly (via
+second moments). The end points of the trail are then given by
+:math:`(xc \pm a*cos(\Theta),yc \pm a*sin(\Theta))`.
+"""
+
+import numpy as np
+
+from lsst.meas.base.pluginRegistry import register
+from lsst.meas.base import SingleFramePlugin, SingleFramePluginConfig
+from lsst.meas.base import FlagHandler, FlagDefinitionList, SafeCentroidExtractor
+from lsst.meas.base import MeasurementError
+
+__all__ = ("SingleFrameNaiveTrailConfig", "SingleFrameNaiveTrailPlugin")
+
+
+class SingleFrameNaiveTrailConfig(SingleFramePluginConfig):
+    """Config class for SingleFrameNaiveTrailPlugin.
+    """
+
+    def setDefaults(self):
+        super().setDefaults()
+
+
+@register("ext_trailedSources_Naive")
+class SingleFrameNaiveTrailPlugin(SingleFramePlugin):
+    """Naive trailed source characterization plugin
+    Parameters
```

Comment:
For readability of the docs here and throughout the python packages, add a blank line between the different sections (e.g. Parameters, Returns) in the doc strings.